client/dashboard/style.scss

```diff
@@ -188,3 +153,29 @@
 		}
 	}
 }
+
+.woocommerce-task-dashboard__container .components-button.is-button,
+.woocommerce-profile-wizard__body .components-button.is-button,
+.woocommerce-page .components-modal__frame .components-button.is-button {
```

Comment:
In Firefox and Safari, it looks like the vertical alignment isn't working for buttons:

<img width="440" alt="Screen Shot 2019-11-26 at 3 36 25 PM" src="https://user-images.githubusercontent.com/10561050/69608915-58739d00-1063-11ea-943c-e49031ab3d84.png">
<img width="466" alt="Screen Shot 2019-11-26 at 3 35 50 PM" src="https://user-images.githubusercontent.com/10561050/69608916-590c3380-1063-11ea-9461-6044434d8bc4.png">